packages/core/src/node/messaging/messaging-contribution.ts

```diff
@@ -122,6 +126,7 @@ export class MessagingContribution implements BackendApplicationContribution, Me
             if (allowed) {
                 this.webSocketServer!.handleUpgrade(request, socket, head, client => {
                     this.webSocketServer!.emit('connection', client, request);
+                    this.messagingListener.websocketDidUpgrade(request, client);
```

Comment:
I double checked with the convention used for such APIs and I was wrong... It should instead be `onDidWebsocketUpgrade`. e.g. 

https://github.com/eclipse-theia/theia/blob/96930a2edcc4c706d41ab8486500de7979d01283/packages/filesystem/src/browser/filesystem-watcher.ts#L160